modules/addons/src/gltf/gltf-instantiator.js

```diff
@@ -38,7 +38,7 @@ export default class GLTFInstantiator {
 
   createAnimator() {
     if (Array.isArray(this.gltf.animations)) {
-      return new GLTFAnimator(this.gltf);
+      return new GLTFAnimator(this.gltf, this.options.timeline, this.options.channel);
```

Comment:
- Channels need to be removed with the glTF scene is removed
- Separate out the glTF stuff to a different PR?

Reply:
The channel isn't added by the animator, so it shouldn't remove it.

Reply:
Separate out the glTF stuff to a different PR